composer.json

```diff
@@ -16,8 +16,13 @@
     ],
     "require": {
         "php": "^8.0.2",
+        "ext-ctype": "*",
+        "ext-filter": "*",
+        "ext-hash": "*",
```

Comment:
Required in:

- `illuminate/auth`
- `illuminate/routing`
- `illuminate/cookie`
- `illuminate/encryption`

Optional in:

- `illuminate/filesystem`
- `illuminate/broadcasting`